logic.py

```diff
@@ -21,13 +21,23 @@ def execute(bot):
 #TODO: Implement these bot methods to bot
 
 class Forward(Order):
-    def execute(bot,distance):
-        bot.moveForward(distance)
+    def execute(bot):
+        bot.moveForward(1)
+        raise NotImplementedError()
+
+class ForwardTwice(Order):
+    def execute(bot):
+        bot.moveForward(2)
+        raise NotImplementedError()
+
+class RotateLeft(Order):
```

Comment:
Don't we want a single `Rotate` abstract class, and you can just have a constant that changes?